jenkins/root-build.cmake

```diff
@@ -696,6 +712,8 @@ function(CONFIGURE_ROOT_OPTIONS)
       set(specflags ${specflags} -DCMAKE_CXX_STANDARD=17)
     elseif("cxx14" IN_LIST SPECLIST)
       set(specflags ${specflags} -DCMAKE_CXX_STANDARD=14)
+    elseif("root7" IN_LIST want_modules)
+      set(specflags ${specflags} -DCMAKE_CXX_STANDARD=14)
```

Comment:
What happens if the build was configured to be C++17, is this going to override it to C++14 then?

Reply:
Seems to be yes. 
I have to add extra check if C++ standard was already specified

Reply:
Finally, c++ standard should be ok.
By default it was forced to c++11.
Now with root7 it will be forced to c++14 (beside centos)
And as before, you could provide -DCMAKE_CXX_STANDARD=17 in your custom flags.
These flags placed at the end of cmake options list and will overrule default settings